package.json

```diff
@@ -80,7 +82,7 @@
         "test:visual:clean:baseline": "rimraf test/visual/screenshots-baseline",
         "test:visual:clean:current": "rimraf test/visual/screenshots-current",
         "test:watch": "yarn test:watch:focus unit",
-        "test:watch:focus": "yarn test:build && run-p watch:css \"test:build -w\" \"test:start --watch --group {1}\" --",
+        "test:watch:focus": "yarn test:build && run-p watch:css build:watch \"test:start --watch --group {1}\" --",
```

Comment:
Curious what the {1} does here?  Is that referencing the $1 input variable for the command or is this something else?

Reply:
It applies the argument from `test:watch:focus tooltip` to that specific command `test:start --watch --group tooltip`. So, maybe the same? I had never done it before this and the blog I read said jump, and I asked how high 😉